upload/src/addons/SV/ThreadReplyBanner/_output/templates/public/SV_ThreadReplyBanner.html

```diff
@@ -1,15 +1,15 @@
-<xen:if is="{$thread.banner}">
-	<xen:require css="panel_scroller.css" />	
-	<xen:require css="ThreadReplyBanner.css" />
-	<div class="PanelScroller ThreadReplyBanner">
-		<div class="scrollContainer">
-			<div class="PanelContainer">
-				<ol class="Panels">
-					<li class="panel Notice DismissParent">
-						<div class="baseHtml noticeContent">{xen:raw $thread.banner}</div>
-					</li>
-				</ol>
-			</div>
-		</div>
-	</div>
-</xen:if>
\ No newline at end of file
+<xf:if is="{{
+	(
+		$xf.visitor.hasPermission('forum','sv_replybanner_show') ||
+		$xf.visitor.hasPermission('forum','sv_replybanner_manage')
+	) &&
+	($bannerNotices && $thread.ThreadBanner.banner_state)
+}}">
```

Comment:
As mentioned previously, use `$thread.canViewBanner()` to-do the permission check and then `$thread.ThreadBanner.getBanner()` (or similar named) to get the data for calling the notice_list.